icionmine/src/views/Graph.tsx

```diff
@@ -6,7 +6,7 @@ import './styles/homeStyle.css';
 class Home extends React.Component<{}, {}>  {
```

Comment:
Don't hesitate to make the names clearer. For example, you can specify what it is as a graphic and for what